src/components/backend-ai-resource-policy-list.ts

```diff
@@ -741,10 +762,10 @@ export default class BackendAIResourcePolicyList extends BackendAIPage {
   }
 
   /**
-   * Set a TextEl value according to toggle checkbox checked state.
-   *
-   * @param {Event} e - Dispatches from the native input event each time the input changes.
-   */
+  * Set a TextEl value according to toggle checkbox checked state.
+  *
+  * @param {Event} e - Dispatches from the native input event each time the input changes.
+  */
```

Comment:
Is there a reason or a style guide to account for these changes? I am usually accustomed to the convention with spaced comments for readability: https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/#aligning-comments.

Also, this kind of style principle should be mechanically checked by activating the ES linter, which might be a good separate refactoring issue.

Reply:
Good point. I'm not sure the current linting rule will automatically fix this without changing other codes strange, for this time I'll fix it manually.